### ISSUE TITLE
The iis_app :config action is not documented in the README.MD file.

### ISSUE THREAD

**Issue by ericrlarson** [chef-cookbooks/iis#237]
The iis_app :config action is not documented in the README.MD file.

It should be mentioned there.  If I had not happened to be reading the code I would not have known that I can have the resource configure an existing application.

**Comment by EasyAsABC123** [comment]
it appears to be documented right here:
https://github.com/chef-cookbooks/iis#iis_app

**Comment by ericrlarson** [comment]
No it is not.  Here is what I see.

```
Actions

:add - add a new application pool
:delete - delete an existing application pool

Attribute Parameters
```

The add and delete actions are there but the config action is not.

**Comment by ericrlarson** [comment]
If you look at the code here https://github.com/chef-cookbooks/iis/blob/7886cad1d94bd232184e9d23765ec7129ad2ad40/providers/app.rb you will see that the config action is implemented.

**Comment by EasyAsABC123** [comment]
Directly from the readme.md file
```
iis_app
--------

Creates an application in IIS.

### Actions

- `:add` - add a new application pool
- `:delete` - delete an existing application pool

### Attribute Parameters

- `site_name` - name attribute. The name of the site to add this app to
- `path` -The virtual path for this application
- `application_pool` - The pool this application belongs to
- `physical_path` - The physical path where this app resides.
- `enabled_protocols` - The enabled protocols that this app provides (http, https, net.pipe, net.tcp, etc)

### Example

```ruby
# creates a new app
iis_app "myApp" do
  path "/v1_1"
  application_pool "myAppPool_v1_1"
  physical_path "#{node['iis']['docroot']}/testfu/v1_1"
  enabled_protocols "http,net.pipe"
  action :add
end
```

**Comment by EasyAsABC123** [comment]
resolved in #d323559cfd2f4743b9ca188a21a420f906816af2
